app/src/main/java/com/github/doomsdayrs/apps/shosetsu/backend/TarArchive.kt

```diff
@@ -0,0 +1,83 @@
+package com.github.doomsdayrs.apps.shosetsu.backend
+
+import org.kamranzafar.jtar.*
+import java.io.File
+import java.io.InputStream
+import java.io.RandomAccessFile
+
+class TarArchive(val file: File)
+
```

Comment:
For this file,

Open it in Android studio then press the following shortcut:  
Alt+Shift+L to reformat the file.

once it's well formatted we can focus on improving the semantics and API.